src/lib/es5.d.ts

```diff
@@ -4311,6 +4311,7 @@ declare namespace Intl {
         style: string;
         currency?: string;
         currencyDisplay?: string;
+        currencySign?: string;

```

Comment:
Shouldn't this, and `currencyDisplay`, go in the [es2020.intl.d.ts](https://github.com/microsoft/TypeScript/blob/master/src/lib/es2020.intl.d.ts#L264-L278) interface instead?

From MDN [Compatibility Chart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#browser_compatibility) it looks like this was released at the same time as all of the other options listed in the es2020 interface.

Reply:
@DanielRosenwasser [previously commented](https://github.com/microsoft/TypeScript/pull/40876#issuecomment-702463480) that Intl isn't part of the ecmascript standard, so I think it should actually stay in ES5. Daniel, can you confirm?

Reply:
We want this PR as it is, es2020 is the right pattern - these APIs may not tie directly to the esXXXX specs but they are released and supported in a similar way to how we structure these libs. This came out in 2020 and we should reflect that via the `target` - which this PR does.